oeps/oep-0002.rst

```diff
@@ -69,6 +70,32 @@ following fields:
     If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be
     ``False``, unless the reporting tool can auto-detect accordance.
 
+``track-pulls``: boolean (optional)
+
+    .. This should be moved to a separate OEP
+
+    Whether or not this repository should be tracked for creation of OSPR tickets.
```

Comment:
Agree. Sounds like we should rip this out and revisit if that use case ever emerges. Seems easy enough to bolt on later if we need it.